renderdoc/driver/metal/official/metal-cpp.cpp

```diff
@@ -0,0 +1,33 @@
+/******************************************************************************
```

Comment:
Could not decide if should put this in "official" or "3rdParty".
I chose "official" because it is owned by Apple and I am treating it as Apple official C++ Metal interface.

Reply:
I like official better, since yes this feels like it should be treated as spec headers. The distinction is somewhat nebulous but 'official' tends to be for anything which defines an API or is otherwise driver- or platform- specific, whereas '3rdparty' tends to be for external libraries or utilities that are more generic or solve a task rather than provide an interface.